gateway/api/views/jobs.py

```diff
@@ -99,35 +99,32 @@ def get_queryset(self):
         return queryset
 
     @_trace
-    @action(methods=["POST"], detail=True)
-    def add_runtime_jobs(
+    @action(methods=["GET", "POST"], detail=True)
+    def runtime_jobs(
         self, request, pk=None
     ):  # pylint: disable=invalid-name,unused-argument
-        """Add RuntimeJob to Job"""
-        if not request.data.get("runtime_job"):
-            return Response(
-                {
-                    "message": "Got empty `runtime_job` field. Please, specify `runtime_job`."
-                },
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+        """Handle RuntimeJob objects associated to Job"""
         job = self.get_object()
-        runtimejob = RuntimeJob(
-            job=job,
-            runtime_job=request.data.get("runtime_job"),
-            runtime_session=request.data.get("runtime_session"),
-        )
-        runtimejob.save()
-        message = "RuntimeJob is added."
-        return Response({"message": message})
 
-    @_trace
-    @action(methods=["GET"], detail=True)
-    def list_runtime_jobs(
-        self, request, pk=None
-    ):  # pylint: disable=invalid-name,unused-argument
-        """List RuntimeJob objects associated to Job"""
-        job = self.get_object()
-        runtimejobs = job.runtime_jobs.all()
-        serializer = RuntimeJobSerializer(runtimejobs, many=True)
-        return Response({"runtime_jobs": serializer.data})
+        # POST: associate runtime jobs to serverless job
+        if request.method == "POST":
```

Comment:
The good thing is that, in a next PR, when we move this end-point under v1 following the new architecture, we are going to be able to separate this logic in two files. For me it's fine as it is by now. Thanks!